src/main/java/com/datastax/cdm/job/BaseJobSession.java

```diff
@@ -24,8 +24,8 @@ public abstract class BaseJobSession {
     // then do the following to set the values as they are only applicable per JVM
     // (hence spark Executor)...
     // Rate = Total Throughput (write/read per sec) / Total Executors
-    protected RateLimiter readLimiter;
-    protected RateLimiter writeLimiter;
+    protected RateLimiter originLimiter;
```

Comment:
#nit should we name these as `originReadRateLimiter` and `targetWriteRateLimiter` for clarity?